src/shared/components/header/index.js

```diff
@@ -7,13 +7,20 @@ import DesktopNavbar from 'shared/components/navbar/desktop'
 import MobileNavbar from 'shared/components/navbar/mobile'
 import styles from './index.module.css'
 
+const defaultScrollOptions = { offset: -66, align: 'top', duration: 800 }
+
 class Header extends Component {
   state = {
-    isSticky: false
+    isSticky: false,
+    gettingStartedElement: undefined
+  }
+
+  componentDidMount () {
+    this.scrollToComponent = require('react-scroll-to-component')
```

Comment:
Why is this being done here? If you are having problems in the build process is because you need to do a conditional require after the import statements:

```js
const scrollToComponent = typeof window !== 'undefined' && require('react-scroll-to-component');
```

But try with a regular import first.